binary_and_decimal.rb

```diff
@@ -12,6 +12,7 @@ def binary_to_decimal
       decimal_total += 2**i
     end
   end
+  #  puts "The binary number #{binary_num} = #{decimal_total}."
   return decimal_total
 end
 
```

Comment:
If the arrays are not equal in length, the method should return false since in that case the arrays won't be containing all the same elements - one of them will have more elements.